UnitySDK/Assets/ML-Agents/Scripts/Constants.cs

```diff
@@ -0,0 +1,11 @@
+namespace MLAgents
+{
+    /// <summary>
+    /// Grouping for use in AddComponentMenu (instead of nesting the menus).
+    /// </summary>
+    public enum MenuGroup
```

Comment:
Better way to do this / better place to put it?

Reply:
Maybe `EditorConstants` and put it in the Editor folder.